doc/src/examples/algorithms/azimuth.cpp

```diff
@@ -0,0 +1,42 @@
+// Boost.Geometry
+// QuickBook Example
+
+// Copyright (c) 2021, Oracle and/or its affiliates
+// Contributed and/or modified by Adam Wulkiewicz, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+//[azimuth
+//` Shows how to calculate azimuth
+
+#include <iostream>
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/point_xy.hpp>
+
+int main()
+{
+    typedef boost::geometry::model::d2::point_xy<double> point_type;
```

Comment:
We could use `using` - but it's fine anyway. Minor.

Reply:
If `using` is prefered we should probably do this consistently in all examples at once.